UploadOrganizationLogo/index.ts

```diff
@@ -30,6 +32,8 @@ winston.add(contextTransport);
 const app = express();
 secureExpressApp(app);
 
+app.use(bodyParser.json({ limit: "50mb" }));
```

Comment:
is this a reasonable limit? (what's the expected max size? the current one?)

Reply:
Navigating through [io-services-metadata](https://github.com/pagopa/io-services-metadata/tree/master/logos) it seems that the expected max size is 1 Mb. With a conservative approach we can define a limit up to 5-10 MB.

Reply:
yes, can we update to 5mb?

Reply:
ok 👍